libdevcore/TrieDB.h

```diff
@@ -462,16 +462,13 @@ class FatGenericTrieDB: private SpecificTrieDB<GenericTrieDB<_DB>, h256>
 		iterator() { }
 		iterator(FatGenericTrieDB const* _trie): Super(_trie) { }
 
-		typename Super::value_type at() const
+		bytes key() const
```

Comment:
I was not right, it is used in State also to have a map of addresses to storage and we need to iterate easily over it getting addresses (not hashes)

I will better add a second type of iterator here